autoload/dirvish.vim

```diff
@@ -495,7 +495,7 @@ func! s:open_dir(d, reload) abort
   if a:reload || s:should_reload()
     call s:buf_render(b:dirvish._dir, get(b:dirvish, 'lastpath', ''))
     " Set up Dirvish before any other `FileType dirvish` handler.
-    exe 'source '.fnameescape(s:srcdir.'/ftplugin/dirvish.vim')
+    runtime! ftplugin/dirvish.{vim,lua}
```

Comment:
well, this is contrary to the comment immediately above this line. The reason for the current approach was so that ftplugins could override/delete whatever Dirvish's ftplugin does.

Reply:
oh, thanks, i saw the comment now :)
it actually does not need to check `if b:did_ftplugin` here, as `vim-dirvish/ftplugin/dirvish.vim` already checks this variable. then i see no meaning for this pr and i dont think #299 should be considered to be a bug.

Reply:
(a) https://github.com/justinmk/vim-dirvish/blob/master/ftplugin/dirvish.vim#L1-L4